presto-docs/src/main/sphinx/sql/create-table.rst

```diff
@@ -17,6 +17,9 @@ Description
 Create a new, empty table with the specified columns.
 Use :doc:`create-table-as` to create a table with data.
```

Comment:
Sorry about that correct, I thought you added support for `IF NOT EXIST` to `create-table-as` also. In case you this support, please change the doc accordingly.